meta-lmp-base/recipes-support/mfgtool-files/mfgtool-files_0.1.bb

```diff
@@ -9,6 +9,7 @@ INITRAMFS_IMAGE = "fsl-image-mfgtool-initramfs"
 DEPENDS = "${INITRAMFS_IMAGE}"
 
 UUU_RELEASE = "1.3.154"
+FULL_FLASH_IMAGE ?= "factory-image"
```

Comment:
As we discussed on slack, I would prefer having this set to an image that is available in meta-lmp alone, such as lmp-base-console-image. We can change ci-scripts (bb-config) to set a default value for factory users, as we already do for image name.

Then it can be easily customized by our factory user as well (in case they decide to produce an image with a different name).

Reply:
Agreed and done.